addons/mass_mailing/models/mailing.py

```diff
@@ -67,6 +67,7 @@ def _get_default_mail_server_id(self):
 
     active = fields.Boolean(default=True, tracking=True)
     subject = fields.Char('Subject', help='Subject of your Mailing', required=True, translate=True)
+    preview = fields.Char('Preview', help='Preview of emails to send. Will be added in an invisible element at the beginning of the email (optional).', translate=True)
```

Comment:
I was thinking something a bit more verbose for our poor users that may not be super knowledgeable with that stuff.
And also less technical, they don't care that the preview is added "in an invisible element", it might be very confusing for them.

Sugg:
> Catchy preview sentence for the email (optional).
> Most modern mailing clients will display that preview next to the subject before opening the email.
> (This text will not be part of the email content.)